packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -87,8 +107,15 @@ export class RetriableDocumentStorageService implements IDocumentStorageService
                 }
                 // If it is not retriable, then just throw the error.
                 if (!canRetryOnError(err)) {
+                    this.logger.sendErrorEvent({
+                        eventName: `StorageCallsNonRetriable_${fetchCallName}`,
+                        numRetries,
+                        totalTime: Date.now() - startTime,
```

Comment:
should this be a perf event, rather than a custom event with total time?

Reply:
Umm, I don't think we want to always record performance here. I am just logging the event in case we retried or an error event if it was non retriable error. The individual calls performance will be recorded in driver itself. So for most of the calls where we would be successful in first try, I don't think it would be beneficial. 
That said, is there a way to not log a performance event in case we didn't retried? Cancelling will log a cancel event.
Thoughts?